mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/StoreDocumentTournament.kt

```diff
@@ -134,11 +114,83 @@ class StoreDocumentTournament(
               .whereEquals("roundDepth", round)
               .get<ChessDocument>()
               .toPoolResults()
-      createMatchesForRankedPlayers(results) { (round - 1).takeIf { it >= 1 } }
+      store.transaction { createMatchesForPoolResults(results) { (round - 1).takeIf { it >= 1 } } }
+    }
+  }
+
+  private fun Transaction<DocumentReference>.createPools(
+      tournamentDocumentReference: DocumentReference,
+      current: TournamentDocument,
+      maxPlayers: Int,
+      poolSize: Int,
+      bestOf: Int,
+      adminId: String,
+      players: List<String>
+  ) {
```

Comment:
Missing documentation.

Reply:
Anyway, fixed in https://github.com/epfl-SDP/android/pull/392/commits/a8685cd7be6104b3a4f8315aabff2c43f026a76b